limesurvey/settings/production.py

```diff
@@ -0,0 +1,16 @@
+"""
+Settings for the LimeSurvey plugin.
+"""
+
+def plugin_settings(settings):
+    """
+    Read / Update necessary project settings for production envs.
+    """
```

Comment:
you're missing these ones: https://github.com/eduNEXT/xblock-limesurvey/pull/14/files#diff-e619c549b6432b92a13bf224f0c40e2ee268eedbcbf2175b418826854dd79187R11-R13